test/lib/ufe/testMaterial.py

```diff
@@ -88,7 +88,7 @@ def testUsdNoMaterial(self):
         materials = materialInterface.getMaterials()
         self.assertEqual(len(materials), 0)
 
-        if(os.getenv('UFE_PREVIEW_VERSION_NUM', '0000') >= '5003'):
+        if(os.getenv('UFE_MATERIAL_HAS_HASMATERIAL', 'NOT-FOUND') in ('1', 'TRUE')):
```

Comment:
Used the wrong logic. The cmake variable will have "TRUE" or "FALSE" in it. So you cannot just test for the existence of the var - it will always exist. Followed logic of another test.